modules/configuring/proc-provisioning-your-custom-configuration.adoc

```diff
@@ -20,8 +20,12 @@ It contains one secret per line in `KEY=value` form.
 
 . Author your custom `{my-app-config-file}` file.
 This is the main {product-short} configuration file.
-+
-The `baseUrl` field is mandatory in your `{my-app-config-file}` file to ensure proper functionality of {product-short}. You must specify the `baseUrl` in both the `app` and `backend` sections to avoid errors during initialization.
+
+** To prepare a deployment with the {product} Operator on OpenShift, you can start with an empty file.
+Creating a custom file, even an empty one, is required to avoid the {product} Operator to revert user edits during upgrades.
+
+** To prepare a deployment with the {product} Helm chart, or on Kubernetes, add the `baseUrl` field in your `{my-app-config-file}` file to ensure proper functionality of {product-short}.
+Specify the `baseUrl` in both the `app` and `backend` sections to avoid errors during initialization.
```

Comment:
Can you also mention the `backend.cors.origin` field? This is equally important as well, as it might otherwise have an impact on using RHDH from the browser.

Reply:
Done